### PR TITLE
add html task in html folder

### PR DESCRIPTION
**Pull request by joymeetme**
### Task: <task name here>

I <clear and concise explanation>
I <clear and concise explanation>

### CODE REVIEW

**Comment by kenny-wealth** [comment]
> Task: 
> 
> I 
> I

What were you trying to do here @joymeetme ?

File: submissions/ayodeleayobami/html/HTML TASK.html [hunk 1]
```diff
@@ -0,0 +1,31 @@
+<!Doctype html>
+<html lang="en">
+    <head>
+        <title>perfectfit Login Page</title>
+        <meta charset="utf-8">
+        <meta name="viewport" content="width=device-width initial-scale=1.2">
+    </head>
+    <body>
+        
+<h2>Welcome to perfectfit,this is an online shopping store for shoes, we provide comfort to
```
Comment (kenny-wealth) [inline submissions/ayodeleayobami/html/HTML TASK.html]: Fix indentation here...

File: submissions/ayodeleayobami/html/HTML TASK.html [hunk 2]
```diff
@@ -0,0 +1,31 @@
+<!Doctype html>
```
Comment (kenny-wealth) [inline submissions/ayodeleayobami/html/HTML TASK.html]: Please rename file to have no spaces. Refer to [Styleguide here](https://github.com/Dufuna-CodeCamp/Dufuna-CodeCamp19/blob/master/STYLEGUIDE.md
)

Comment (kenny-wealth) [reply submissions/ayodeleayobami/html/HTML TASK.html]: "Doctype" should be all lowercase. Refer to Styleguide

File: submissions/ayodeleayobami/html/HTML TASK.html [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+            <fieldset>
+                <perfectfit>Welcome</perfectfit>
+                   <label for="username">Username</label><br>
+                   <input type="text" name="username" id="username" placeholder= "enter your username" required><br><br>
+                   <label for="email">Email</label><br>
+                   <input type="text" name="Email" id="email" placeholder= "enter your email" required><br><br>
+                   <label for="password">Password</label><br>
+                   <input type="password" name="password" id="password" placeholder="******" min="6" required><br><br>
+                   <input type="checkbox" name="keepme" value="">
+                   <label for="loginmode">Remember me</label>
```
Comment (kenny-wealth) [inline submissions/ayodeleayobami/html/HTML TASK.html]: There is no form control named "loginmode". What does this label point to?

**Review by joymeetme — COMMENTED** [review]
made corrections as requested.

File: submissions/ayodeleayobami/html/task.html [hunk 4]
```diff
@@ -0,0 +1,28 @@
+<!doctype html>
+<html lang="en">
+    <head>
+        <meta charset="utf-8">
+        <meta name="viewport" content="width=device-width initial-scale=1.2">
+        <title>perfectfit Login Page</title>
+    </head>
+    <body>       
+<h2>Welcome to perfectfit,this is an online shopping store for shoes, we provide comfort to
```
Comment (tegaphilip) [inline submissions/ayodeleayobami/html/task.html]: You started with `<h2>` but closed with `</h4>` . You should choose one.

File: submissions/ayodeleayobami/html/task.html [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+        <meta name="viewport" content="width=device-width initial-scale=1.2">
+        <title>perfectfit Login Page</title>
+    </head>
+    <body>       
+<h2>Welcome to perfectfit,this is an online shopping store for shoes, we provide comfort to  
+           your feet and make you look classy and chic.
+        </h4>
+        <form method="POST" action="login.php">
+            <fieldset>
+                <perfectfit>Welcome</perfectfit>
```
Comment (tegaphilip) [inline submissions/ayodeleayobami/html/task.html]: Is there a special html tag called `perfectfit`?

**Review by joymeetme — COMMENTED** [review]
made corrections.

**Review by tegaphilip — DISMISSED** [review]
@joymeetme good job

**Review by joymeetme — COMMENTED** [review]
added css to html task.

**Review by tegaphilip — APPROVED** [review]

**Review by kenny-wealth — APPROVED** [review]

**Pull request merged**